### DISCUSSION TITLE
Welcome to nhs.pycom Discussions!

### DISCUSSION THREAD

**Discussion by craig-shenton** [nhs-oa-community/nhs.pycom#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by craig-shenton** [comment]
Welcome @maryamanuelnhsx @Arouba 👋

**Reply by maryamanuel1** [reply]
thanks @craig-shenton!

**Comment by jamie10** [comment]
Hi, I'm Sam (but have different username). I am building up my knowledge of python to use conjointly with other analytical software as part of my role as Senior Analyst in NHSEI. I'm keen to link in with this group, thanks for setting up!

**Reply by maryamanuel1** [reply]
Welcome Sam! We're at the early stages of development but If you would like to join our MS teams group, please send us your email to pycom@nhsx.nhs.uk.

Mary

**Reply by jamie10** [reply]
sure thing, will drop an email now :)

**Reply by maryamanuel1** [reply]
Hey @jamie10  - you can sign to nhs.pycom here: https://docs.google.com/forms/d/e/1FAIpQLSe74CGuc-xaJ-ccHRiNqHVNC8Iw4TYMLXfSSaWdj6SxFQr5wQ/viewform - we're sending out the first newsletter soon :)

**Comment by drcjar** [comment]
Greetings @jamie10 and all! I'm an nhs respiratory consultant and pythonista, pleased to see this initiative! have booked to come to the meetup https://www.events.england.nhs.uk/events/pycom-the-nhs-python-community and looking forward to it!

best, Carl

**Reply by Arouba** [reply]
Thank you Carl, we look forward to seeing you!

**Reply by maryamanuel1** [reply]
Hey Carl - you can sign to nhs.pycom here: https://docs.google.com/forms/d/e/1FAIpQLSe74CGuc-xaJ-ccHRiNqHVNC8Iw4TYMLXfSSaWdj6SxFQr5wQ/viewform - we're sending out the first newsletter soon :)

**Comment by heritage-cmd** [comment]
My name is Timi. I recently completed a Python course and I am hoping to make a great impact with it in NHS. I currently work as a Clinical Trial Assistant (Data Assistant where I am involved in the Data management for Clinical Research) and I will be starting a new role next week as the Data Lead for the Critical Care Department in one of the NHS Trust in West Midlands. I am not confident with the usage of Python yet but I am hoping if I could gain insights from other members of this group.

**Reply by maryamanuel1** [reply]
Hey Timi/@heritage-cmd, thanks so much for introducing yourself and congrats on the new job! 

We are currently registering interest through this survey, and will be in touch with members with an update on progress soon! https://docs.google.com/forms/d/e/1FAIpQLSe74CGuc-xaJ-ccHRiNqHVNC8Iw4TYMLXfSSaWdj6SxFQr5wQ/viewform

Kind regards,

Mary

**Comment by syedameersohail** [comment]
Hi @craig-shenton , I am Ameer Sohail, I am Data Analyst at NECS . I would be interested and would love to present on events related to on Machine Learning, Natural Language Processing with Python.

Best Regards,

Ameer
